src/Command/SyncRecipesCommand.php

```diff
@@ -80,6 +88,45 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $operations[] = new InstallOperation($pkg);
         }
 
+        if ($createEnvLocal = $force && file_exists($this->rootDir.'/.env') && file_exists($this->rootDir.'/.env.dist') && !file_exists($this->rootDir.'/.env.local')) {
+            $io->writeError('Renaming .env -> .env.local and .env.dist -> .env');
```

Comment:
Because this is SUCH a specific case, add a link? https://symfony.com/doc/current/configuration/dot-env-changes.html